pkg/api/pluginproxy/pluginproxy_test.go

```diff
@@ -142,7 +169,7 @@ func getPluginProxiedRequest(ctx *models.ReqContext, cfg *setting.Cfg, route *pl
 	}
 	proxy := NewApiPluginProxy(ctx, "", route, "", cfg)
 
-	req, err := http.NewRequest(http.MethodGet, route.URL, nil)
+	req, err := http.NewRequest(http.MethodGet, "/api/plugin-proxy/grafana-simple-app/api/v4/alerts", nil)
```

Comment:
What's the purpose of this change? Just asking because I'd like to understand :)

Reply:
Update: Never mind, I found your reply to @marefr earlier clearing this up. I think an explanatory code comment would be helpful here though.